content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,46 @@ See the Application Security [set up documentation][11] to validate you you are
 * If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false`, and restart your service.
-* If ASM was activated in one click: go to [ASM service configuration][15], select **Services with Threat Management Capabilities**, click on your service, and then click **Deactivate**.
-* To disable ASM on your services in bulk, click the check box in the list header and then under **Bulk Actions** select **Deactivate Threat Management detection on (number of) services**.
+* If threat management was activated using [Remote Configuration][16], do the following: 
+  1. Go to [Services][15] (**ASM** > **Catalog** > **Services**).
+  2. Select **Threat Management in Monitoring Mode**.
+  3. In the **Threat Management** facet, enable **Monitoring Only**, **No data**, and **Ready to block**.
+  4. Click on a service.
+  5. In the service details, in **Threat Detection**, click **Deactivate**.
 
-## Disabling Software Composition Analysis (SCA)
+<div class="alert alert-info">If threat management was activated using <a href="https://app.datadoghq.com/organization-settings/remote-config">Remote Configuration</a>, you can use a <strong>Deactivate</strong> button. If threat management was activated using local configuration, the <strong>Deactivate</strong> button is not an option.</div>
+
+* To disable threat management on your services in bulk, do the following: 
+  1. Go to [Services][15].
+  2. In the **Threat Management** facet, enable **Monitoring Only**, **No data**, and **Ready to block**.
+  3. Select the check boxes for the services where you want to disable threat detection.
+  4. In **Bulk Actions**, select **Deactivate Threat detection on (number of) services**.
+
+## Disabling Software Composition Analysis
 
 To disable [Software Composition Analysis][14]:
 
-* Go to [ASM service configuration][15], select **Services with Open Source Vulnerability Detection**, click on your service and then click **Deactivate**.
-* To disable Software Composition Analysis on your services in bulk, click the check box in the list header and then under **Bulk Actions** select **Deactivate Open source vulnerability detection on (number of) services**.
+* Go to [Services][15], select **Software Composition Analysis (SCA)**, click on your service and then click **Deactivate**.
+* To disable Software Composition Analysis on your services in bulk, click the check box in the list header and then under **Bulk Actions** select **Deactivate Software Composition Analysis (SCA) on (number of) services**.
 
 ## Disabling Code Security
 
 <div class="alert alert-info">Code Security vulnerability detection is in beta.</div>
 
 To disable [Code Security][13] vulnerability detection, remove the `DD_IAST_ENABLED=true` environment variable from your application configuration, and restart your service. 
 
+If no `DD_IAST_ENABLED=true` environment variable is set for your service, do the following:
```

Comment:
We don't have IAST on PHP so having this documentation could cause confusion.

Reply:
@Anilm3 I can clarify it like this:

> To disable [Code Security][13] vulnerability detection, remove the `DD_IAST_ENABLED=true` environment variable from your application configuration, and restart your service. This does not apply to PHP apps.

Sound good?

Reply:
Yes, the clarification is sufficient in my opinion.